app/liquid_tags/github_tag/github_readme_tag.rb

```diff
@@ -4,38 +4,35 @@ class GithubTag
   class GithubReadmeTag
     PARTIAL = "liquids/github_readme".freeze
 
+    attr_reader :client, :content, :options, :readme_html
```

Comment:
Just added some readers to use throughout the file.

Reply:
they can probably be private right?

Reply:
They can, though I often don't bother with readers. Happy to change it though if you prefer.

Reply:
nope, it's fine :)